src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -132,9 +154,13 @@ export const PositionsTriggersCell = ({
                   <Button
                     action={ButtonAction.Primary}
                     size={ButtonSize.Small}
-                    onClick={() => null} // TODO: CT-663 Implement onClick functionality
+                    onClick={openTriggersDialog}
                   >
-                    {stringGetter({ key: STRING_KEYS.EDIT_STOP_LOSS })}
+                    {stringGetter({
+                      key: isStopLossOrder(order)
+                        ? STRING_KEYS.EDIT_STOP_LOSS
+                        : STRING_KEYS.EDIT_STOP_LOSS, // TODO: CT-704 update to EDIT_TAKE_PROFIT
```

Comment:
my bad here, forgot a string key 😢 made a mini task to update this later, didn't want to hold up the PR on this